docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -196,11 +336,242 @@ To process the received command on the server side:
+   dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Server state: %d\n", server_name, server_state);
+   ```
+
+3. Send the command to the server using the corresponding `mc_client_send_XXX_cmd()`. Use the server name retrieved in the previous step to identify the server.
+
+   For example, to send a playback action change command to the server, use the `mc_client_send_playback_action_cmd()` function with the new action defined in the third parameter:
+
+   ```
+   mc_playback_action_e playback_action = MC_PLAYBACK_ACTION_PLAY;
+   char *request_id = NULL;
+
+   ret = mc_client_send_playback_action_cmd(g_client_h, server_name, playback_action, &request_id);
+   ```
+
+   If you want to define your own commands to send to the server, use the `mc_client_send_custom_cmd()` function.
+   The request_id will be passed to the `mc_client_cmd_reply_recieved_cb()`.
+
+4. Destroy the media controller client handle using the `mc_client_destroy()`, when media controller client handle is no longer needed:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+To process the received command on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Define the callback that is invoked when the server receives the command.
+
+   For example, to define a callback for playback state change commands:
+
+   ```
+   void
+   playback_action_cmd_received_cb(const char* client_name, const char *request_id, mc_playback_action_e action, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request Id: %s, Playback action: %d\n", client_name, request_id, action);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for playback state change commands, use the `mc_server_set_playback_action_cmd_received_cb()` function.
+   - To register a callback for playback position change commands, use the `mc_server_set_playback_position_cmd_received_cb()` function.
+   - To register a callback for shuffle mode change commands, use the `mc_server_set_shuffle_mode_cmd_received_cb()` function.
+   - To register a callback for repeat mode change commands, use the `mc_server_set_repeat_mode_cmd_received_cb()` function.
+   - To register a callback for played item, playback state and playback position change commands in playlist, use the `mc_server_set_playlist_cmd_received_cb()` function.
+   - To register a callback for a custom command, use the `mc_server_set_custom_cmd_received_cb()` function.
+
+   For example, to register a callback for playback state change commands:
+
+   ```
+   ret = mc_server_set_playback_action_cmd_received_cb(g_mc_server, playback_action_cmd_received_cb, NULL);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()`, when media controller server handle is no longer needed:
 
    ```
    mc_server_destroy(g_server_h);
    ```
+
+<a name="send_cmd_reply"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The sentense is not header, so I changed it to header and The HTML tag changed to head anchor for MD.